src/server.c

```diff
@@ -5042,6 +5046,8 @@ sds genRedisInfoString(const char *section) {
             "active_defrag_misses:%lld\r\n"
             "active_defrag_key_hits:%lld\r\n"
             "active_defrag_key_misses:%lld\r\n"
+            "total_active_defrag_exceeded_time:%lld\r\n"
+            "current_active_defrag_exceeded_time:%lld\r\n"
```

Comment:
maybe "total_frag_exceeded_time"?
i.e. the fragmentation is over the threshold..
or maybe "total_defrag_exceeded_time"?